### PR TITLE
[Tizen Studio] 4.5.1 Release Notes

### PR DESCRIPTION
**Pull request by manishtr**
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

4.5.1 Tizen Studio Release Notes

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
Looks good to me

**Comment by mijin85cho** [comment]
Hello. Would you please review this PR with high priority?

As written in the release note, we have to publish this page till Dec 29, this Wednesday.

@sangwook96 If the target release date is changed, please update this PR also.

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - Tizen Studio does not support the [Smart Development Bridge](../common-tools/smart-development-bridge.md) (SDB) bash auto-completion on Windows (it is available on Ubuntu and macOS).
+- Dynamic Analyzer
+  - When analyzing applications on commercial devices running Tizen 3.0, newly-released or after a firmware update, the following problems exist:
+    - The Core Frequency information is not shown.
+    - The screenshots on the scene transitions feature will not work.
+  - When analyzing applications on the Tizen 4.0 emulator or reference device, the startup profiling information is not shown.
+  - The UI Hierarchy viewer feature and startup profiling are not performed simultaneously.
+  - The Dynamic Analyzer cannot show lifecycle information for Web applications.
+  - Widget applications cannot be profiled with the Dynamic Analyzer. They are hidden in the application list on the toolbar for all Tizen platforms, except Tizen 2.3.2.
+  - The Dynamic Analyzer sometimes gets stuck which is caused by an internal security checking program.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:57]:
```suggestion
  - The Dynamic Analyzer sometimes gets stuck, this is caused by an internal security checking program.
```

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- VSCode
+  - Hybrid applications are not supported yet.
+- Visual Studio
+  - Hybrid applications are not supported yet.
+  - NUI XAML application build error can be resolved by doing clean and then build again.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:62]: Meaning is not clear, please consider revising the sentence. 
```suggestion
  - NUI XAML application build error can be resolved by doing clean and then build again.
```

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- VS Mac
+  - NUI XAML application build error can be resolved by doing clean and then build again.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:64]: Meaning is not clear, please consider revising the sentence. 
```suggestion
  - NUI XAML application build error can be resolved by doing clean and then build again.
```

File: docs/application/tizen-studio/release-notes/release-notes.md [hunk 4]
```diff
@@ -1 +1 @@
-4-5-release-notes.md
\ No newline at end of file
+4-5-1-release-notes.md
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/release-notes.md:1]: This is not showing up in the Left Navigation Panel.
image

**Review by asm-foysal — COMMENTED** [review]
@manishtr, hello! Please address the comments made in this review.

File: docs/application/tizen-studio/release-notes/release-notes.md (hunk 4 above)
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/release-notes.md:1]: Its Visible to me, request you to please check it once more

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  - To use Tizen Emulator, use Intel VTx supported CPU and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for Tizen Emulator, see [Prerequisites for Tizen Studio](../setup/prerequisites.md).
+    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows, you must set the Tizen Emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; control panel.
+    - On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session occasionally logs out while launching Emulator Manager, or the emulator skin is displayed improperly. Verify the prerequisites and upgrade to the latest graphics driver.
+  - On Windows, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
```
[the remark this answers is not in the document]
Comment (asm-foysal) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:38]: This still needs to be addressed.

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md (hunk 2 above)
Comment (asm-foysal) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:62]: This still needs to be addressed.

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md (hunk 3 above)
Comment (asm-foysal) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:64]: This still needs to be addressed.

**Review by asm-foysal — COMMENTED** [review]
@manishtr can you please address these concerns:

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:38]: the display surface might not be rendered correctly if the emulator window is obstructed.

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md (hunk 2 above)
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:62]: This means to do clean build again

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md (hunk 3 above)
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:64]: This means to do clean build again

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  - To use Tizen Emulator, use Intel VTx supported CPU and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for Tizen Emulator, see [Prerequisites for Tizen Studio](../setup/prerequisites.md).
+    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows, you must set the Tizen Emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; control panel.
+    - On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session might occasionally log out while launching Emulator Manager or the emulator skin may be displayed improperly. If this happens, verify the prerequisites and upgrade to the latest graphics driver.
+  - On Windows, depending on your OS theme (such as Non-Aero themes and Windows XP themes), display surface rendering might be suspended if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:38]:
```suggestion
  - On Windows, depending on your OS theme (such as Non-Aero themes and Windows XP themes), display surface rendering might be suspended if the emulator window is covered by another window. If you click the emulator window, the display surface should run correctly again.
```

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  - When analyzing applications on the Tizen 4.0 emulator or reference device, the startup profiling information is not shown.
+  - The UI Hierarchy viewer feature and startup profiling are not performed simultaneously.
+  - The Dynamic Analyzer cannot show lifecycle information for Web applications.
+  - Widget applications cannot be profiled with the Dynamic Analyzer. They are hidden in the application list on the toolbar for all Tizen platforms, except Tizen 2.3.2.
+  - The Dynamic Analyzer sometimes gets stuck, this is caused by an internal security checking program.
+- VSCode
+  - Hybrid applications are not supported yet.
+- Visual Studio
+  - Hybrid applications are not supported yet.
+  - NUI XAML application build error can be resolved by doing clean build.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:62]:
```suggestion
  - NUI XAML application build error can be resolved by doing a clean build.
```

File: docs/application/tizen-studio/release-notes/4-5-1-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+- VS Mac
+  - NUI XAML application build error can be resolved by doing clean build.
```
Comment (asm-foysal) [inline docs/application/tizen-studio/release-notes/4-5-1-release-notes.md:64]:
```suggestion
  - NUI XAML application build error can be resolved by doing a clean build.
```

**Review by asm-foysal — COMMENTED** [review]
@manishtr thanks for clearing these up, I've provided suggestions with the appropriate corrections where necessary. 

@mijin85cho once these minor grammatical errors are addressed, this PR should be ready for approval.

**Comment by manishtr** [comment]
> @manishtr thanks for clearing these up, I've provided suggestions with the appropriate corrections where necessary.
> 
> @mijin85cho once these minor grammatical errors are addressed, this PR should be ready for approval.

Thanks @asm-foysal for the review, the changes requested have been incorporated

**Review by asm-foysal — APPROVED** [review]
@manishtr Thank you for incorporating all the changes.

@mijin85cho I'm approving this PR for merging.

**Comment by mijin85cho** [comment]
@asm-foysal Thanks for the review.

@manishtr Will the release be done as the planned today(Release Date: Dec 29, 2021)?
If there is no update, this PR should be merged.

**Pull request merged**